app/views/assessments/verification/verification-1.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_tag("#{request.fullpath}", method: "post") do %>
+      <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            <div class="govuk-caption-l">Verification checks</div >
+            <h1 class="govuk-fieldset__heading">
```

Comment:
I think this el is missing an `id` attribute, is needed for link in validation/error message

Reply:
We'll come back to this, having bumped the question of links-in-error messages to @laurencecodes.

Reply:
Outcome of this was to just make the error message display text instead of a link